### PR TITLE
newsletter: Dec 2019

### PR DESCRIPTION
**Pull request by orrabin**
@tbrisker Please take a look, we might want to add more to the 1.24 notes or a whole section about Foreman 2.0 coming up next.

### CODE REVIEW

File: _posts/2019-12-25-foreman-community-newsletter---december-2019.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+See you there at the booth, Foreman room and our traditional
+hackday on the 5th! Tickets are available. 
+
+### [More Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+All Foreman events are available via [the forum
+calendar](https://community.theforeman.org/calendar) - hit the Subscribe button
+(which will give you an ICS link). You can also use the "Add To Calendar" button
+on specific events if you don't wish to import the whole thing.
+
```
Comment (ohadlevy) [inline _posts/2019-12-25-foreman-community-newsletter---december-2019.md:66]: I didn't see any links from the calendar? perhaps it make sense to link from here to our calendar (or to the official documentation of  the events)...

Comment (orrabin) [reply _posts/2019-12-25-foreman-community-newsletter---december-2019.md:66]: I added the events to the calendar, I can switch the links here as well.

File: _posts/2019-12-25-foreman-community-newsletter---december-2019.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+### Conferences
+
+[DevConf.CZ](https://www.devconf.info/cz/): 24-26 January in Brno.
+We'll be there at the Foreman stand.
+Tickets are running out fast so go and get one!
+
+[FOSDEM](https://fosdem.org/2020): 1-2 February in Brussels.
+Come visit us at the booth or drop by on one of the Foreman talks
```
Comment (tbrisker) [inline _posts/2019-12-25-foreman-community-newsletter---december-2019.md]:
```suggestion
Come visit us at the booth on floor 2 of the K building or drop by on one of the Foreman talks
```

**Review by tbrisker — APPROVED** [review]
thanks @orrabin !

**Pull request merged**